packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -31,6 +31,13 @@ export 'package:flutter/gestures.dart' show
 ///
 /// Attempts to recognize gestures that correspond to its non-null callbacks.
 ///
+/// GestureDetector also listens for accessibility events and maps
+/// them to the callbacks. A tap will result in onTap, a long press in
+/// onLongPress, a vertical drag in either onVerticalDragUpdate or
+/// onPanUpdate, and a horizontal drag in either
+/// onHorizontalDragUpdate or onPanUpdate. The corresponding down/up
+/// or start/end events do _not_ fire in this case.
```

Comment:
That seems slightly dangerous.  Can the existing clients actually cope with that?